lib/locales/br/emails/added-as-project-member.hbs

```diff
@@ -1,6 +1,9 @@
 <h1>Você está em uma equipe!</h1>
-<p>Você foi adicionado(a) ao <strong>{{projectName}}</strong>  como membro do projeto para o desafio <strong>{{challengeName}}</strong>. Para obter mais detalhes, clique no botão abaixo.</p>
-<a href="{{challengeUrl}}" class='cta-button'>PÁGINA DO DESAFIO</a>
+<p>Você foi adicionado(a) ao
+  <strong>{{projectName}}</strong>
+  como membro do projeto para o desafio
+  <strong>{{challengeName}}</strong>. Para obter mais detalhes, clique no botão abaixo.</p>
+<a href='{{challengeUrl}}' class='cta-button'>PÁGINA DO DESAFIO</a>
 <p>Se você tiver algum problema, copie esta URL e a cole em seu navegador:</p>
-<a href="{{challengeUrl}}">{{challengeUrl}}</a>
+<a href='{{challengeUrl}}'>{{challengeUrl}}</a>
```

Comment:
Is that okay to use single quotes to wrap a href in HTML?

Reply:
Yes, it's okay and there should not be any problems. 

There is a convenience in React where HTML quotes are in `double quotes`, while it's `single quotes` elsewhere.